### PR TITLE
Add shellcheck to basic features

### PR DESCRIPTION
**Pull request by meatballhat**
- [x] bump to 0.4.5

### CODE REVIEW

File: cookbooks/travis_build_environment/attributes/default.rb [hunk 1]
```diff
@@ -154,6 +154,11 @@
 default['travis_build_environment']['sysctl_disable_ipv6'] = true
 default['travis_build_environment']['wget']['version'] = '1.18'
 
+default['travis_build_environment']['shellcheck_url'] = 'https://s3.amazonaws.com/travis-blue-public/binaries/ubuntu/14.04/x86_64/shellcheck-0.4.4.tar.bz2'
+default['travis_build_environment']['shellcheck_version'] = '0.4.0'
```
Comment (ljharb) [inline cookbooks/travis_build_environment/attributes/default.rb]: this is 0.4.0 but the URL is for 0.4.4, and the latest version is 0.4.5. Could we bump them all to 0.4.5?

Comment (meatballhat) [inline cookbooks/travis_build_environment/attributes/default.rb]: Egad!  Thanks for the catch!  Yes, working on a 0.4.5 binary now :+1:

**Comment by meatballhat** [comment]
@ljharb Not by default.  I'd have to backport it 😐 .  I'd rather not backport it.  Precise is going away 🔜, although I expect we'll keep it around after EOL ... for some amount of time...

**Comment by meatballhat** [comment]
Workin' https://travis-ci.org/meatballhat/cloaked-octo-tyrion/builds/182436175

**Pull request merged**